examples/meta/CMakeLists.txt

```diff
@@ -121,6 +121,7 @@ INSTALL(DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/cpp
 IF (NOT EXISTS ${CMAKE_CURRENT_BINARY_DIR}/data)
     # copy data symlink to cmake binary dir so that we can execute meta examples in there
     CREATE_DATA_SYMLINK(${CMAKE_CURRENT_SOURCE_DIR}/data ${CMAKE_CURRENT_BINARY_DIR}/data)
+    CREATE_DATA_SYMLINK(${CMAKE_SOURCE_DIR}/data ${CMAKE_CURRENT_BINARY_DIR}/full-data)
```

Comment:
I either do this, or move uci optdigits dataset to toy data

Reply:
Nope can’t do that

Toy means toy, so pls don’t put the full dataset but rather a small version of it, or better use an existing one (these are just api examples they’re supposed to run fast)

Reply:
I tried some of the classification datasets, and the results were bad, so I added a smaller version of this dataset.

Reply:
Just checked, clustering the 2d nonlinear classification example should work, as this is just a 2x2 gaussian bumps grid. If not, something is wrong :)

Reply:
I get around 0.5 accuracy :"(

Reply:
which algorithm on what data?

Reply:
KMeans clustering on `classifier_binary_2d_nonlinear_features_train.dat`